### DISCUSSION TITLE
Examples of Storefront API uses

### DISCUSSION THREAD

**Discussion by morganmccunn** [Shopify/storefront-api-feedback#9] [Show & Tell]
We're often amazed at the beautiful and powerful experiences our community builds for their customers. If you're using the Storefront API or related SDKs to power your shopping experience, please share! 👀 

Sometimes it's about what you _can't see._ If you have a tech stack you'd like to share, this is the place for it too. 😍 

Here are some awesome examples we've put together for you:
* [Babylist case study](https://www.shopify.ca/enterprise/headless-commerce-vs-traditional-commerce)
* [KOTN case study](https://www.shopify.com/plus/customers/kotn)
* [ComplexLand case study](https://www.shopify.com/plus/customers/complex-networks)

**Comment by dotcollective-brendon** [comment]
Sharing our current tech stack for headless solutions:

- Shopify (commerce)
- Sanity (content)
- Firebase (hosting, functions, auth, db)
- Gatsby (frontend)
- Other: Typescript / Tailwind / Chakra / Apollo

Also a few builds:

- https://bared.com.au/
- https://www.myride.com.au/
- https://skinjuice.com.au/
- https://rossiboots.com.au/
- https://kyhastudios.com/
- https://crepprotect.com.au/
- https://www.mitchellandness.com.au/

**Reply by ahmed-adly-khalil** [reply]
These are really great stores, these are static gatsby sites hosted on firebase?

**Reply by dotcollective-brendon** [reply]
@ahmed-adly-khalil Correct! static Gatsby, context API for global state management, firebase for hosting, auth (social login), firestore (realtime db with live listeners for features like wishlist) and functions (multipass etc..)

**Reply by ahmed-adly-khalil** [reply]
Awesome, you have a very talented team by the way. 
any thoughts on the experience with firebase in terms of cost compared to traditional hosting or Vercel like options?

**Reply by dotcollective-brendon** [reply]
Indeed, we are fortunate to have such a great team!

The majority of sites fall into the free tier usage which is quite generous, the largest that runs flash sales are under $200 a month.

**Comment by magnetism2000** [comment]
Nice @b0123498765 !

We are currently running this stack and working nicely:

- Storefront API
- Vue Storefront 2
- Vercel (using vercel cache)
- Prismic (content)
- Algolia
- Tailwind

https://bareblends.com.au

**Comment by maddalax** [comment]
Our current stack is:

- Storefront API
- Typescript
- NextJS (React)
- Tailwind CSS
- Seperate NodeJS API
- DigitalOcean Managed Kubernetes for deploying API & Storefront

https://eattoevolve.com/

**Comment by arobbins** [comment]
Stack:
- Storefront API
- Admin API
- User connects via unlisted Shopify app
- WordPress Plugin

https://wpshop.io
